kubejs/server_scripts/base/tags/items/forge/storage/sophisticated_storage.js

```diff
@@ -0,0 +1,10 @@
+onEvent('item.tags', (event) => {
+    event.add('forge:storage/basic/wooden','#sophisticatedstorage:base_tier_wooden_storage','sophisticatedstorage:basic_tier_upgrade','sophisticatedstorage:shulker_box')
```

Comment:
I did that mostly for quests about sophisticated storage, but to not make those only "for quests purposes" I tried making them actually somewhat useful.
If you want I can change those to `sophisticatedstorage:` and change a bit naming, in result making those tags tier own group.

Reply:
I think that'd be better, yes please.

Reply:
Alrighty so script is done, tag names changed to storage_{tier}
I would normally use mod name, but it would be a pain to write this entire mod name everytime you want to search some iron tiered chests 😅 If you have better idea for a tag name, I'm open for suggestion